ifrs17/Test/QueriesTest.ipynb

```diff
@@ -590,6 +597,84 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "code",
+            "source": [
+                "var args = new Args(\"CH\",2020,9,Periodicity.Monthly,null);",
+                "\nvar testData = new YieldCurve[] {new YieldCurve{ Currency = \"EUR\", Year = 2020, Month = 6, Name = \"A\", Values = new double[]{0.1,0.2,0.3,0.4,0.5,0.6} }, ",
```

Comment:
make this case more interesting by adding some YC with Name = null
a Name A within 2016 and 2020 to update CurrentPrevious result

Reply:
done

Reply:
done